daffodil-runtime1/src/main/java/org/apache/daffodil/api/validation/ValidatorsFactory.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.api.validation;
+
+import com.typesafe.config.Config;
+import org.apache.daffodil.lib.validation.DaffodilLimitedValidator$;
+import org.apache.daffodil.lib.validation.NoValidator$;
+import org.apache.daffodil.lib.validation.XercesValidatorFactory;
+
+import scala.collection.immutable.Seq;
+
+import java.net.URI;
+import java.util.Collections;
+
+public class ValidatorsFactory {
+  private ValidatorsFactory() {
+  }
+
+  public static Validator getNoValidator() {
```

Comment:
This getNoValidator() is for internal use. It just returns a static object, 

Remove this method so it is not in the API. Everyplace it is called we just use the scala `NoValidator` object since it is only called from scala code.